### PR TITLE
[Utopia-402] new home page

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- add new home page for ppq

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [ ] N/A
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [ ] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

File: src/frontend/src/components/common/Navbar/navPages.ts
Comment (antsand) [inline src/frontend/src/components/common/Navbar/navPages.ts:3]: What is the significance of id: 2? I would try to avoid constants and have a more clear description what this is for.

File: src/frontend/src/components/common/Navbar/index.tsx
Comment (antsand) [inline src/frontend/src/components/common/Navbar/index.tsx:18]: Is this only applicable to even pages? What does 'even' pages mean? I would avoid this type of check. adding a class 'ms-auto' to this check is not very clear.

File: src/frontend/src/pages/PPQPage/PPQPage.scss
Comment (Bottle7) [inline src/frontend/src/pages/PPQPage/PPQPage.scss:48]: If the only thing that this class will do is apply  `font-weight: 700`, then lets add it to a higher-level file and name it something more generic that aligns with our CSS Standards. Documentation on these standards may be found in our wiki.

File: src/frontend/src/sass/_variables.scss
Comment (Bottle7) [inline src/frontend/src/sass/_variables.scss:14]: Is this colour used anywhere else? If not, maybe we should leave it out of our branding variables.

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (kushal-arora-fw) [inline src/frontend/src/pages/PPQPage/PPQPage.tsx:26]: no empty strings at the end

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (kushal-arora-fw) [inline src/frontend/src/pages/PPQPage/PPQPage.tsx:24]: No hardcoded text. Please use a separate message file. Refer any `messages.ts` for formatting.

File: src/frontend/src/sass/_variables.scss
Comment (binggaofw) [reply src/frontend/src/sass/_variables.scss:14]: so far, I only find we use it here. but I think we will use it later so i will put it in our branding variables

**Review by kushal-arora-fw — CHANGES_REQUESTED** [review]

File: src/frontend/src/sass/_variables.scss
Comment (Bottle7) [reply src/frontend/src/sass/_variables.scss:14]: Based on conversations with Jordan and Anthony, we will only be using colours and elements as defined by the Figma files under 'Components.' The UI/UX team has assured us that they will only be using styling as defined under that section.

File: src/frontend/src/components/common/Navbar/index.tsx
Comment (binggaofw) [reply src/frontend/src/components/common/Navbar/index.tsx:18]: Menu
it follows the design, make sure one in left and another one in right for navbar item

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (Bottle7) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx:18]: Another alternative would be to not define such specific classes unless absolutely necessary.

Ideally, we should be able to cover 90-95% of use cases with generic global classes / nested selectors as this will provide the most consistent look and feel for the application.

File: src/frontend/src/sass/_variables.scss
Comment (antsand) [reply src/frontend/src/sass/_variables.scss:14]: Please DO NOT add any colours that are not there in the branding component page.

File: src/frontend/src/components/common/Navbar/index.tsx
Comment (antsand) [reply src/frontend/src/components/common/Navbar/index.tsx:18]: There is a better way to do this. I would avoid this type of logic. 
Please have one class and make sure it fits both designs. With one button or two button and it should render correctly.

File: src/frontend/src/sass/_variables.scss
Comment (binggaofw) [reply src/frontend/src/sass/_variables.scss:14]: this color actually is focus, i just remove this one and use focus

File: src/frontend/src/components/common/Navbar/navPages.ts
Comment (binggaofw) [reply src/frontend/src/components/common/Navbar/navPages.ts:3]: change back

File: src/frontend/src/components/common/Navbar/index.tsx
Comment (binggaofw) [reply src/frontend/src/components/common/Navbar/index.tsx:18]: just remove this

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx:26]: done

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx:24]: done

File: src/frontend/src/components/common/Navbar/index.tsx
Comment (binggaofw) [reply src/frontend/src/components/common/Navbar/index.tsx:10]: @Bottle7 I am not sure why we need this in navbar file, do you have any idea?

File: src/frontend/src/components/common/Navbar/navPages.ts
Comment (binggaofw) [reply src/frontend/src/components/common/Navbar/navPages.ts:5]: this involve change backend, because backend redirect url point to /ppq, we need to create a new ticket I think

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx:19]: i agree we do not define such specific classes unless we must do it.

**Comment by akroon3r** [comment]
@binggaofw can you review the output from the frontend lint please? I just merged main into your branch after merged 437 into main and found some issues.

**Comment by akroon3r** [comment]
@binggaofw can you review the output from the frontend lint please? I just merged main into your branch after merged 437 into main and found some issues.

File: src/frontend/src/components/common/Navbar/index.tsx
Comment (kushal-arora-fw) [reply src/frontend/src/components/common/Navbar/index.tsx:10]: ahh, you know what my bad! I thought the map function was introduced as part of this change itself. Just realized you just removed the `ul` here. Ignore the above comment, please!

File: src/frontend/src/sass/_form.scss
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_form.scss:4]: possibly extra spacing in line 4.

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (kushal-arora-fw) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx:19]: sure, I'd love that! No unnecessary classes, unless absolutely required.

so, I think we are action pending here @binggaofw, right? I mean if you are getting rid of the unnecessary classes @binggaofw or **if we** absolutely require it, can you rename it appropriately?

File: src/frontend/src/sass/_form.scss
Comment (kushal-arora-fw) [inline src/frontend/src/sass/_form.scss:3]: You do need an *empty* end of line here

Comment (binggaofw) [reply src/frontend/src/sass/_form.scss]: I see what you mean. I turn off the linting function as it will throw an error.  add back

**Review by antsand — COMMENTED** [review]
Let the public landing page commit go first. There are some merge conflicts.

File: src/frontend/src/pages/PPQPage/PPQPage.scss
Comment (antsand) [inline src/frontend/src/pages/PPQPage/PPQPage.scss:1]: We should NOT be defining this like this. There is nothing unique about this style for this container.

We should be thinking about defining classes that apply to the whole application. Not to individual boxes.

Comment (antsand) [reply src/frontend/src/pages/PPQPage/PPQPage.scss:1]: If the box is unique we need to have a conversation with UI/UX. This box is not unique.

**Comment by binggaofw** [comment]
@antsand sure，I can hold this pr until your PR merged first

**Review by antsand — CHANGES_REQUESTED** [review]
We have discussed not to create CSS styles like this. 

Let us have a pair programming session to do this properly.

File: src/frontend/src/pages/PPQPage/PPQPage.scss
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.scss:1]: yes, I just replace this with bcgovPageContainer

Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.scss]: let me know if it works in your local, looks good in my local

File: src/frontend/src/pages/PPQPage/messages.ts
Comment (antsand) [inline src/frontend/src/pages/PPQPage/messages.ts:1]: Just wondering. To create these message are we following some standards? Or it is always custom?

File: src/frontend/src/sass/_buttons.scss
Comment (antsand) [inline src/frontend/src/sass/_buttons.scss:27]: Thank you!!! my bad :(

File: src/frontend/src/sass/_form.scss
Comment (antsand) [inline src/frontend/src/sass/_form.scss:2]: This should be part of the text tags (h1, h2, h3, etc). We should not be defining font-size like this.

File: src/frontend/src/pages/PPQPage/PPQPage.tsx
Comment (antsand) [inline src/frontend/src/pages/PPQPage/PPQPage.tsx:22]: What does this class do that the h2 tag cannot?

Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/PPQPage.tsx:22]: text weight is 700

File: src/frontend/src/sass/_form.scss
Comment (binggaofw) [reply src/frontend/src/sass/_form.scss:2]: I discuss this with @Bottle7. Hi @Bottle7 do you have thoughts?

File: src/frontend/src/pages/PPQPage/messages.ts
Comment (binggaofw) [reply src/frontend/src/pages/PPQPage/messages.ts:1]: @Bottle7 @kushal-arora-fw  any idea?

Comment (antsand) [reply src/frontend/src/pages/PPQPage/messages.ts:1]: I have some ideas. we can discuss later. For now it is fine.

**Review by antsand — APPROVED** [review]

File: src/frontend/src/pages/PPQPage/messages.ts
Comment (kushal-arora-fw) [reply src/frontend/src/pages/PPQPage/messages.ts:1]: Not really a global standard per say. But this is something I have followed in one of my previous projects and has worked out well.

The idea was to remove the usage of string in the jsx files/components, and have standard messaging framework. And we be ready for language switches when we bring localization/internationalization for French into the system.

Ofcourse, I have some enhancements in mind. However, as that can be for future.

**Review by kushal-arora-fw — APPROVED** [review]
Looks good

**Pull request merged**
